source/reference/mongosync.txt

```diff
@@ -296,6 +296,25 @@ error occurs, the ``mongosync`` log may contain the word "error' but
 ``mongosync`` is still able to complete the sync. In the case that a
 sync does not complete, ``mongosync`` writes a fatal log entry. 
 
+UUIDs
+~~~~~
+
+``mongosync`` creates new :abbr:`UUIDs (universally unique identifiers)`
+for collections on the destination cluster. There is no relationship
+between UUIDs on the source cluster and the destination cluster. If
+applications contain hard-coded UUIDs (which MongoDB does not
+recommend), you may need to update the application before it works
+properly with a migrated cluster. 
+
+Sorting
+~~~~~~~
+
+``mongosync`` creates a new natural order (without any sort method
+specified) on the destination cluster. If applications depend on
```

Comment:
mongosync doesn’t “create an order”; rather, the order in which it inserts documents is undefined.
```suggestion
``mongosync`` inserts documents on the destination cluster in an undefined order.
Thus, natural sort order is not preserved from the source to the destination. If applications
depend on
```

Reply:
Done!